src/config/constants/pools.ts

```diff
@@ -16,6 +16,32 @@ const pools: PoolConfig[] = [
     sortOrder: 1,
     isFinished: false,
   },
+  {
+    sousId: 201,
+    stakingToken: tokens.cake,
+    earningToken: tokens.axs,
+    contractAddress: {
+      97: '',
+      56: '0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82',
```

Comment:
This is CAKE token address 😕
And in AXS-CAKE pool its AXS address.
Unless I missed something this should be smart chef contracts

Reply:
You're correct - these are wrong

Reply:
I copied from the Trello, looks like there was a mistake